### PR TITLE
Refactor swift examples for async/await

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

File: app/views/docs/getting-started-for-server.phtml [hunk 1]
```diff
@@ -273,13 +273,8 @@ val response = account
...
 
-let response = account
-    .createJWT() { result in
-        switch result {
-        case .failure(let error): print(error.message)
-        case .success(let token): print(token.jwt)
-        }
-    }</code></pre>
+let response = try await account
+    .createJWT()</code></pre>
```
Comment (Meldiron) [inline app/views/docs/getting-started-for-server.phtml:277]:
```suggestion
let response = try await account.createJWT()</code></pre>
```

File: app/views/docs/getting-started-for-apple.phtml [hunk 2]
```diff
@@ -159,12 +156,8 @@ let client = Client()
 
 // Register User
 let account = Account(client: client)
-account.create(userId: "unique()", email: "email@example.com", password: "password") { result in
-    switch result {
-    case .failure(let error): print(error.message)
-    case .success(let user): print(String(describing: user))
-    }
-}
+let user = account.create(userId: "unique()", email: "email@example.com", password: "password")
```
Comment (Meldiron) [inline app/views/docs/getting-started-for-apple.phtml:159]:
```suggestion
let user = try await account.create(userId: "unique()", email: "email@example.com", password: "password")
```

File: app/views/docs/getting-started-for-server.phtml [hunk 3]
```diff
@@ -488,12 +483,8 @@ println(res.body?.string())</code></pre>
 
 let users = Users(client: client)
 
-users.create(userId: "unique()", email: "email@example.com", password: "password") { result in
-    switch result {
-    case .failure(let error): print(error.message)
-    case .success(let user): print(String(describing: user))
-    }
-}
+val user = users.create(userId: "unique()", email: "email@example.com", password: "password")
```
Comment (Meldiron) [inline app/views/docs/getting-started-for-server.phtml:486]:
```suggestion
val user = try await users.create(userId: "unique()", email: "email@example.com", password: "password")
```

File: app/views/docs/getting-started-for-server.phtml [hunk 4]
```diff
@@ -666,11 +657,8 @@ client
 
 let users = Users(client: client)
 
-users.create(userId: "unique()", email: "email@example.com", password: "password") { result in
-    switch result {
-    case .failure(let error): print(error.message)
-    case .success(let user): print(String(describing: user))
-    }
+let user = users.create(userId: "unique()", email: "email@example.com", password: "password")
```
Comment (Meldiron) [inline app/views/docs/getting-started-for-server.phtml:660]:
```suggestion
let user = try await users.create(userId: "unique()", email: "email@example.com", password: "password")
```

File: app/views/docs/pagination.phtml [hunk 5]
```diff
@@ -219,38 +207,27 @@ class MainActivity : AppCompatActivity() {
...
     let client = Client()
       .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
       .setProject("my_project") // Your project ID
 
     let database = Database(client)
-    // page 1
-    database.listDocuments(
+    
+    // Page 1
+    var list = database.listDocuments(
```
Comment (Meldiron) [inline app/views/docs/pagination.phtml:218]:
```suggestion
    var list = try await database.listDocuments(
```

File: app/views/docs/pagination.phtml [hunk 6]
```diff
@@ -219,38 +207,27 @@ class MainActivity : AppCompatActivity() {
...
-                }
-            }
-        }
-    }
+    )
 
+    let lastId = list.documents[list.documents.count - 1].$id
+            
+    // Page 2
+    list = database.listDocuments(
```
Comment (Meldiron) [inline app/views/docs/pagination.phtml:226]:
```suggestion
    list = try await database.listDocuments(
```

**Review by Meldiron — COMMENTED** [review]
Some things regarding consistency I noticed. Might and might not be what needs to be done in Swift.

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**